src/algorithms/CMakeLists.txt

```diff
@@ -5,3 +5,4 @@ add_subdirectory( tracking )
 add_subdirectory( pid )
 add_subdirectory( digi )
 add_subdirectory( reco )
+add_subdirectory( fardetectors )
```

Comment:
We've been trying to keep algorithms organized by function rather than region. Would this fit in `reco`?

Reply:
Maybe at the moment but it really needs to be sub divided into digi, tracking and reco in the future. I have been using fardetectors as a placeholder for the LowQ2 development too, currently containing an alternative tracking pathway to acts.